client/components/bcat-form-sections/funding-eligibility.tsx

```diff
@@ -43,10 +43,10 @@ export const FundingEligibility: React.FC = () => {
             </div>
 
             <p className='font-bold'>
-              The percentage share of BCAAP funding available for your project may be increased by
-              up to 15% based on whether your community, facility or project meet any of the
-              following criteria (subject to verification by BCAAP staff). Please indicate which, if
-              any, of the following apply, with a brief explanation for those that you so indicate:
+              The percentage share of BCAT funding available for your project may be increased by up
```

Comment:
I think this text should come from the gov team. At the very least they need to review it. We don't know if it's the case BCAT.